viewer/src/datamodels/pointcloud/PointCloudNode.ts

```diff
@@ -92,6 +94,58 @@ export class PointCloudNode extends THREE.Group {
     this._potreeNode.pointShape = value;
   }
 
+  /**
+   * Sets a visible filter on points of a given class.
+   * @param pointClass ASPRS classification class code. Either one of the well known 
+   * classes from {@link WellKnownAsprsPointClassCodes} or a number for user defined classes.
+   * @param visible Boolean flag that determines if the point class type should be visible or not.
+   * @throws Error if the model doesn't have the class given.
+   */
+  setClassVisible(pointClass: number | WellKnownAsprsPointClassCodes, visible: boolean): void {
+    if (!this.hasClass(pointClass)) {
```

Comment:
I think it would be handy to have some method that will return available classes as a subset of WellKnownAsprsPointClassCodes.

Reply:
What would you use this for? I think it's important that we handle all codes equally as it's really just a list of reserved values, other values are also valid.

Reply:
For the filtering in the console. No reason to show all possible (WellKnown) values if there are only a few of them used in a model. But you just said that we can't enumerate all of them anyway if they are custom. So I'm not sure about it anymore.

Reply:
I think we should show all present values in Console. Usually its 5-15 different classes in a file. See http://potree.org/potree/examples/classifications.html for example